### DISCUSSION TITLE
Clapeyron.jl/src/methods /pT.jl

### DISCUSSION THREAD

**Discussion by Kelethabetse** [ClapeyronThermo/Clapeyron.jl#192] [General]
Hello there :)
I want to ask about line 334 and 342 in the following path: [Clapeyron.jl](https://github.com/ClapeyronThermo/Clapeyron.jl/tree/master)/[src](https://github.com/ClapeyronThermo/Clapeyron.jl/tree/master/src)/[methods](https://github.com/ClapeyronThermo/Clapeyron.jl/tree/master/src/methods)
/pT.jl
R is divided by T in both scenarios, I would kindly like to ask if this is a mistake? if not would you kindly elaborate. Thank you :)

**Comment by longemen3000** [comment]
this?
https://github.com/ClapeyronThermo/Clapeyron.jl/blob/4d6dad9a272c1906a16f120f4dc329cedcb537c3/src/methods/pT.jl#L342
it is just a matter of arithmetic order: `x/a/b` is equivalent to `x/(a*b)`. in julia, multiplication and division have the same precedence priority ([Operator Precedence and Associativity](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Operator-Precedence-and-Associativity))

**Reply by Kelethabetse** [reply]
Ahh I see! Thanks so much :)
